TELCODOCS#1255: Added limitations related to creating snapshots and clones in multi-node topology (#69298)
openshift/openshift-docs: modules/lvms-limitations-for-creating-snapshots-and-clones.adoc

// Module included in the following assemblies:
//
// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc

:_mod-docs-content-type: CONCEPT
[id="lvms-limitations-for-creating-snapshots-and-clones_{context}"]
= Limitations for creating volume snapshots and volume clones

{lvms} has the following limitations for creating volume snapshots and volume clones in multi-node topology:

* Creating volume snapshots and volume clones is based on the LVM thin pool capabilities.
* The node must have additional storage after creating a volume snapshot and volume clone for further updating the original data source.

> * Volume snapshots and volume clones can be created only on the node where you have deployed the original data source.

Reviewer: ```suggestion
* Volume snapshots and volume clones get created only on the node where you have deployed the original data source.
```

To avoid passive voice. Please check with SME for technical accuracy though.

Reviewer 2: Meaning is unaffected by this, change is ok from my side

Author: @jakobmoellerdev 
As volume clones and snapshots can be created by users, can we write:

You can create volume snapshots and volume clones only on the node where you have deployed the original data source.

Author: Thank you @jakobmoellerdev, @Srivaralakshmi for the review.
Updated the line.

Author: @Srivaralakshmi -  If we can proceed to merging this PR, could you please add the "peer-review-done" label?
Thanks.

* Pods relying on the persistent volume claim (PVC) that uses the snapshot data and clone data can be scheduled only on the node where you have deployed the original data source.

> * If the PVC is not present and available before creating a volume snapshot or volume clone, then the scheduler in multi-node OpenShift clusters does not provision the volume snapshots and volume clones on the node where you have deployed the original data source.

Reviewer 3: s/OpenShift/{product-title} 

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#product-title-and-version

Reviewer: @bmcelvee Good catch! somehow, I missed this one. TY!

Author: @bmcelvee Thank you for the review.
Updated the content based on your comment.
